spec/requests/content_blocks_spec.rb

```diff
@@ -15,6 +15,8 @@ def compare_content_blocks(object, content_block)
   end
 
   context '#index' do
+    before(:each) { Rails.cache.clear }
```

Comment:
@coorasse we need to test the cache, so it's better to clear the cache instead of disabling it.